### PR TITLE
Buy CryptoCurrency with Transak

### PR DESCRIPTION
**Pull request by guanbinrui**
test

### CODE REVIEW

**Comment by Jack-Works** [comment]
does it ready for review? @guanbinrui

**Comment by Tedko** [comment]
@Jack-Works binrui's waiting for API key on my side. but others should be ready for review

File: src/plugins/Transak/hooks/useTransak.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import TransakSDK, { TransakSDKConfig } from '@transak/transak-sdk'
+import stringify from 'json-stable-stringify'
+import { useMemo } from 'react'
+
+export function useTransak(config?: Partial<TransakSDKConfig>) {
+    const transak = useMemo(
+        () =>
+            new TransakSDK({
```
Comment (Jack-Works) [inline src/plugins/Transak/hooks/useTransak.ts:8]: Does it need to "close" the SDK if config changes?

**Comment by Tedko** [comment]
btw @jk234ert 

Take a look of MetaMask mobile, we can basically do the same:

IMG_8416


38681602925114_ pic

**Comment by Tedko** [comment]
quick question @guanbinrui 
For the next step of your screen can the addr of ETH been autofilled?

File: src/_locales/zh/messages.json [hunk 2]
```diff
@@ -98,6 +98,7 @@
     "activity": "動態",
     "activity_inbound": "收到",
     "activity_outbound": "發出",
+    "buy_now": "",
```
Comment (Tedko) [inline src/_locales/zh/messages.json:101]: "購買"

**Comment by guanbinrui** [comment]
> quick question @guanbinrui
> For the next step of your screen can the addr of ETH been autofilled?

Yep

File: src/plugins/Transak/hooks/useTransak.ts (hunk 1 above)
Comment (guanbinrui) [reply src/plugins/Transak/hooks/useTransak.ts:11]: https://github.com/DimensionDev/Maskbook/pull/1732/files#diff-9153e7f2a7688ef203ed9bb35ea00fa16eea77cad5ec0f501251e41c1c4d155eR30

**Comment by guanbinrui** [comment]
> does it ready for review? @guanbinrui

Ready for review now, but not to merge it.

**Comment by guanbinrui** [comment]
@Jack-Works @septs Ready for review.

**Review by septs — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**